src/main/java/org/truffleruby/cext/ValueWrapperManager.java

```diff
@@ -191,22 +193,32 @@ public synchronized void addFreeBlock(long blockBase) {
 
     public static class HandleBlock {
 
-        public static final HandleBlock DUMMY_BLOCK = new HandleBlock(0, null);
+        public static final HandleBlock DUMMY_BLOCK = new HandleBlock(null, 0, null);
+
+        private static final Set<HandleBlock> keepAlive = new HashSet<>();
```

Comment:
@aardvark179 What do you think of this way to keep them alive?

I think at least we should hold the blocks per-Context (e.g., as a field of ValueWrapperManager), so if a Context is closed the handles could still be released.

Reply:
Yes could be per-context.

Reply:
Could you make it per-Context?

Reply:
I'm inclined to go the simplest thing to start with as I doubt this PR will be our final word on debugging handle problems.

Reply:
OK, @aardvark179 convinced me, since this is purely a debugging option that anyway isn't going to work well in multiple contexts, let's keep it as simple as possible.